app/src/screenshots/res/values/backpack.animation.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<integer name="bpkAnimationDurationXs">0</integer>
```

Comment:
But button pressed animations

Reply:
It does. A lot of APIs ignore that flag.
In our case, the cause is in ButtonStyle.getButtonBackground() method when we manually set enter and exit fade durations from these tokens. It seems it only respects the system values when defaults are not overridden.

I tested this on an emulator with disabled animations and I can confirm that the pressed animation still exists there, unless we override the tokens or don't use it at all.